packages/nx/src/utils/print-help.ts

```diff
@@ -107,7 +107,7 @@ function generateGeneratorOverviewOutput({
       {
         text:
           pluginName +
-          (pluginName.startsWith('@nrwl/')
+          (pluginName.startsWith('@nx/') || pluginName.startsWith('@nrwl/')
             ? chalk.dim(` (v${nxVersion})`)
             : ''),
```

Comment:
Unrelated to the actual change - but does this report a bad version for labs packages like @nrwl/deno?

Reply:
Yes, I think it does.

Reply:
I agree - it also just flat out doesn't report a version for things like nx-dotnet. Here's a fix: https://github.com/nrwl/nx/pull/16353

It does unfortunately conflict with this PR - feel free to hold on merging it until after this one and I can fix it up.